TravelPro.html

```diff
@@ -0,0 +1,301 @@
+            </section>
+        </article>
+
+        <article class="service">
+            <img src="./images/event.png" alt="event">
+            <section class="service-description">
+                <h3>Local Events</h3>
+                <p>barton vanity itself do in it Preferred to men it engrossed listening</p>
+            </section>
+        </article>
+
+        <article class="service">
+            <img src="./images/customization.png" alt="custom">
+            <section class="service-description">
+                <h3>Customization</h3>
+                <p>we deliver outsourced aviation services to our customers</p>
+            </section>
+        </article>
+    </section>
+
+
+
+    <section class="destination-heading">
+        <p>Top Selling</p>
+        <p class="heading">Top Destinations</p>
+    </section>
+   
+
+
+    <section class="destinations">
+        <article class="city">
+            <img src="./images/rome.png" alt="Rome">
+            <section class="city-name">
+                <p>Rome, Italy</p>
+                <p>$5,42k</p>
+            </section>
+            <section class="trip-details">
+                    <img src="./images/send_black.png" height="20px" width="20px">
+                    <span>10 Days Trip</span>
+            </section>
+        </article>
+
+        <article class="city">
+            <img src="./images/london.jpg" alt="London"> 
+            <section class="city-name">
+                <p>London, UK</p>
+                <p>$4.2k</p>
+            </section>
+            <section class="trip-details">
+                    <img src="./images/send_black.png" height="20px" width="20px">
+                    <span>12 Days Trip</span>
+            </section>
+        </article>
+
+        <article class="city">
+            <img src="./images/europe.png" alt="Europe">
+            <section class="city-name">
+                <p>Full Europe</p>
+                <p>$28k</p>
+            </section>
+            <section class="trip-details">
+                    <img src="./images/send_black.png" height="20px" width="20px">
+                    <span>28 Days Trip</span>
+            </section>
+        </article>
+    </section>
+
+
+
+
+    <section class="booking">
+        <section class="left-description">
+            <p>Easy and Fast</p>
+            <p class="book">Book Your Next Trip In 3 Easy Steps</p>
+            <section class="three-steps">
+                <section class="choose">
+                    <section class="icon">
+                        <img class="background" src="./images/sqaure.png" height="25px" width="25px">
+                    </section>
+                    <section class="descript">
+                        <p class="head">Choose Destination</p>
+                        <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Urna Totus Tempus</p>
+                    </section>
+                </section>
+                <section class="payment">
```

Comment:
Using `<section>` for every container, especially for smaller parts like images or buttons, might not be semantically appropriate. Instead, consider using `<div>` for general containers or groupings